zerver/migrations/0285_remove_realm_google_hangouts.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.13 on 2020-06-14 01:58
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+VIDEO_CHAT_PROVIDERS = {
+    'jitsi_meet': {
+        'name': "Jitsi Meet",
+        'id': 1,
+    },
+    'google_hangouts': {
+        'name': "Google Hangouts",
+        'id': 2,
+    },
+}
+
+def remove_google_hangouts_provider(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    # We are removing the Google Hangout integration because Google has
+    # removed the Hangouts brand. All the realms that used Hangouts as
+    # their video chat provided are now setted to the default, jitsi.
+    Realm = apps.get_model('zerver', 'Realm')
+    realm = Realm.objects.filter(video_chat_provider=VIDEO_CHAT_PROVIDERS['google_hangouts']['id'])
+
+    for e in realm:
+        e.video_chat_provider = VIDEO_CHAT_PROVIDERS['jitsi_meet']['id']
+        e.save(update_fields=['video_chat_provider'])
```

Comment:
Let's do this with a single bulk `.update` query, and also not use a singular local variable `realm` for a list.  I think it can be just this with some line-wrapping:

```
 Realm.objects.filter(video_chat_provider=VIDEO_CHAT_PROVIDERS['google_hangouts']['id']).update(video_chat_provider=VIDEO_CHAT_PROVIDERS['jitsi_meet']['id'])`
```